cmd/livegrep-github-reindex/main.go

```diff
@@ -140,17 +140,14 @@ func main() {
 	if err := writeConfig(config, configPath); err != nil {
 		log.Fatalln(err.Error())
 	}
-	if *flagNoIndex {
-		log.Printf("Skipping indexing after writing config")
-		return
-	}
 
 	index := flagIndexPath.Get().(string)
 
 	args := []string{
 		"--out", index,
 		"--codesearch", *flagCodesearch,
 		"--num-workers", *flagNumRepoUpdateWorkers,
+		"--no-index", *flagNoIndex,
```

Comment:
You need to format this bool as a string here -- that's where the CI failure is coming from.